packages/mdc-chips/README.md

```diff
@@ -40,25 +40,52 @@ npm install --save @material/chips
 
 ```html
 <div class="mdc-chip-set">
-  <div class="mdc-chip">
+  <div class="mdc-chip" tabindex="0">
     <div class="mdc-chip__text">Chip content</div>
   </div>
-  <div class="mdc-chip">
+  <div class="mdc-chip" tabindex="0">
     <div class="mdc-chip__text">Chip content</div>
   </div>
-  <div class="mdc-chip">
+  <div class="mdc-chip" tabindex="0">
     <div class="mdc-chip__text">Chip content</div>
   </div>
 </div>
 ```
 
+#### Leading and Trailing Icons
+
+You can optionally add a leading icon (i.e. thumbnail) and/or a trailing icon to a chip. To add an icon, add an `i` element with your preferred icon, give it a class of `mdc-chip__icon`, and a class of either `mdc-chip__icon--leading` or `mdc-chip__icon--trailing`. If you're adding a trailing icon, also set `tabindex="0"` and `role="button"` to make it accessible by keyboard and screenreader.
+
+##### Leading icon
+
+```html
+<div class="mdc-chip mdc-chip--with-leading-icon">
```

Comment:
Remove `--with-leading-icon` here and `--with-trailing-icon` below since they were re-engineered out of existence